src/stream/src/executor/sink.rs

```diff
@@ -55,17 +55,18 @@ async fn build_sink(
 }
 
 // Drop all the UPDATE/DELETE messages in this chunk.
-fn force_append_only(chunk: StreamChunk, data_types: Vec<DataType>) -> StreamChunk {
+fn force_append_only(chunk: StreamChunk, data_types: Vec<DataType>) -> Option<StreamChunk> {
     let mut builder = DataChunkBuilder::new(data_types, chunk.cardinality() + 1);
```

Comment:
Not a big problem by why `+ 1`? 👀

Reply:
Some implementation details: `DataChunkBuilder` will return a chunk once the number of rows reaches its size. But instead of this early return, we want it to return at the end of this function. This `+1` is to avoid this early return.